gateways/orbital/orbital_test.go

```diff
@@ -0,0 +1,245 @@
+// +build unit
+
+package orbital
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/BoltApp/sleet"
+	"github.com/BoltApp/sleet/common"
+	sleet_t "github.com/BoltApp/sleet/testing"
+	"github.com/go-test/deep"
+	"github.com/go-xmlfmt/xmlfmt"
+	"github.com/google/go-cmp/cmp"
+	"github.com/jarcoal/httpmock"
+)
+
+var credentials Credentials = Credentials{"username", "password", 1}
+
+func TestNewClient(t *testing.T) {
+	t.Run("Dev environment", func(t *testing.T) {
+		want := &OrbitalClient{
+			host:        "https://orbitalvar1.chasepaymentech.com/authorize",
+			httpClient:  common.DefaultHttpClient(),
+			credentials: Credentials{"username", "password", 1},
+		}
+
+		got := NewClient(common.Sandbox, Credentials{"username", "password", 1})
+
+		if !cmp.Equal(*got, *want, sleet_t.CompareUnexported) {
+			t.Error("Client does not match expected")
+			t.Error(cmp.Diff(want, got, sleet_t.CompareUnexported))
+		}
+	})
+
+	t.Run("Production environment", func(t *testing.T) {
+		want := &OrbitalClient{
+			host:        "https://orbital1.chasepaymentech.com/authorize",
+			httpClient:  common.DefaultHttpClient(),
+			credentials: Credentials{"username", "password", 1},
+		}
+
+		got := NewClient(common.Production, Credentials{"username", "password", 1})
+
+		if !cmp.Equal(*got, *want, sleet_t.CompareUnexported) {
+			t.Error("Client does not match expected")
+			t.Error(cmp.Diff(want, got, sleet_t.CompareUnexported))
+		}
+	})
+}
+
+func TestSend(t *testing.T) {
+	helper := sleet_t.NewTestHelper(t)
+
+	url := "https://orbitalvar1.chasepaymentech.com/authorize"
+
+	var headerReceived http.Header
+	var requestRaw, responseRaw []byte
+
+	responseRaw = helper.ReadFile("test_data/captureResponse.xml")
+	requestRaw = helper.ReadFile("test_data/captureRequest.xml")
+
+	requestRaw = bytes.TrimSpace(requestRaw)
+
+	base := sleet_t.BaseCaptureRequest()
+	request := buildCaptureRequest(base, credentials)
+
+	request.Body.OrbitalConnectionUsername = "username"
+	request.Body.OrbitalConnectionPassword = "password"
+	request.Body.MerchantID = 1
+
+	t.Run("With Successful Response", func(t *testing.T) {
+		httpmock.Activate()
+		defer httpmock.DeactivateAndReset()
+
+		httpmock.RegisterResponder("POST", url, func(req *http.Request) (*http.Response, error) {
+			headerReceived = req.Header
```

Comment:
lets test headers too - we can do in a separate PR?

Reply:
sounds good